pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -136,6 +136,11 @@ type ComplianceScanType string
 // When changing the defaults, remember to change also the DefaultRawStorageSize and
 // DefaultStorageRotation constants
 type RawResultStorageSettings struct {
+	// Specifies if the raw result storage is disabled. This is useful in case
+	// the raw results are not needed. Defaults to false.
+	// +kubebuilder:validation:Default=false
+	// +kubebuilder:default=false
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
Something I try and do for readability is to prefer the "enabled" case instead of the "disabled" case.

Here, that might mean using `Enabled=true` as the default instead of `Disabled=false`.

When we get into logic latter, it turns:

```go
  if !Disabled {
      // do the enabled thing
  }
```

Into:

```go
  if Enabled {
      // do the enabled thing
  }
```